x-pack/plugin/core/src/main/java/org/elasticsearch/license/XPackLicenseState.java

```diff
@@ -205,12 +209,19 @@
     private static String[] logstashAcknowledgementMessages(OperationMode currentMode, OperationMode newMode) {
         switch (newMode) {
             case BASIC:
-                switch (currentMode) {
-                    case TRIAL:
-                    case STANDARD:
-                    case GOLD:
-                    case PLATINUM:
-                        return new String[] { "Logstash will no longer poll for centrally-managed pipelines" };
+                if (!isBasic(currentMode)) {
```

Comment:
do you mind switching this and other usages to `isBasic(currentMode) == false`? The elasticsearch team has a preference for this over `!`

Reply:
Fixed in d0f05d6.